### ISSUE TITLE
 Anime and Manga show 0 episodes when searching

### ISSUE THREAD

**Issue by SoumyBhow** [jmir1/aniyomi-mpv-beta#4]
Hello. I sincerely apologize if I'm not supposed to give issues on this "buggy" build, but I noticed that if I search for an anime or manga and select it from any extension in the search results, it shows up with 0 episodes/chapters. I've attached an image to show what I mean. 
![Screenshot_20220618-181324_Aniyomi](https://user-images.githubusercontent.com/42043870/174458666-a677dde3-a103-4106-8e7f-b6c7cd607ae1.png)

This happens with every anime/manga in every extension.

**Comment by jmir1** [comment]
Yes, I'm aware of this.
You need to go back and open the manga again. Then the chapters should show up.

**Comment by sinnghabhay80** [comment]
> Yes, I'm aware of this.
> You need to go back and open the manga again. Then the chapters should show up.

Yes, but although it works it would be nice if it would on the first try as it was working before.

**Comment by SoumyBhow** [comment]
> Yes, I'm aware of this. You need to go back and open the manga again. Then the chapters should show up.

Oh I wasn't aware. Selecting again seems to work for now. Though I guess I'll leave this issue open untill it's completely resolved if that's alright. Thank you.

**Comment by Mr-Nobody1** [comment]
Going back doesn't work for me. I have to restart the whole application to do that once I add the title to my library.

**Comment by SoumyBhow** [comment]
I have a question. Are future issues on this mpv build preferred to be made on the original Aniyomi repo, or are they better off here?

**Comment by jmir1** [comment]
> I have a question. Are future issues on this mpv build preferred to be made on the original Aniyomi repo, or are they better off here?

I would say they can stay here.

by the way, this issue whould have been fixed by the latest release. But the episode/chapter list won't "live update" anymore...

**Comment by sinnghabhay80** [comment]
So does this mean this thing is here to stay?

**Comment by SoumyBhow** [comment]
> > I have a question. Are future issues on this mpv build preferred to be made on the original Aniyomi repo, or are they better off here?
> 
> I would say they can stay here.
> 
> by the way, this issue should have been fixed by the latest release. But the episode/chapter list won't "live update" anymore...

Thank you very much for fixing this issue. I can confirm it no longer happens on the latest build. But I don't understand what you mean by it no longer doing "live update". Would you mind clarifying that for me?

**Comment by jmir1** [comment]
@sinnghabhay80 the stable version will eventually be replaced by this beta version, but that might take some time as this is still riddled with bugs 😅 

@SoumyBhow when you mark a chapter/episode as completed, it won't immediately show up in the list.
you need to exit out or scroll far down and up again....
but it's not a big issue

**Comment by sinnghabhay80** [comment]
> So does this mean this thing is here to stay?

What I meant was will live update return? Or is it gone for good?

**Comment by sinnghabhay80** [comment]
Oh thanks for letting me know 😊
